tests/integration/replication-4.tcl

```diff
@@ -103,7 +103,7 @@ start_server {tags {"repl external:skip"}} {
             wait_for_ofs_sync $master $slave
             exec kill -SIGSTOP [srv 0 pid]
             $master incr k
-            after 1000
+            after 1001
```

Comment:
Off by one millisecond. That's means it fails on a really fast machine, not valgrind etc. If you think this is why it fails, then sure, merge it!

Reply:
i'm not sure...
the failure wasn't on valgrind, but it was on GH actions (which are not very speedy).
i don't have any other theory at hand. feel free to suggest one.

Reply:
There is no harm merging this. Then we'll see if it appears again...